src/main/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAO.java

```diff
@@ -160,6 +163,14 @@ public void delete(int id) {
         );
     }
 
+    public void deleteOldDrafts() {
+        jdbcTemplate.update(
+            "DELETE FROM draft_document "
+                + "WHERE updated + interval '1 day' * max_age < now()",
```

Comment:
Is there an index on `updated`? If not, we could add it. Not sure if it matters (because the DBMS might be smart about those things), but maybe index-wise it would be safer to move '1 day' to the other side, i.e. 
```
    WHERE updated max_age < now() - interval '1 day'
```

There's also a question of whether we want to delete drafts last updated 'up until 24th August 2017' or 'up until 24th August 2017 12:34:56.789'. Maybe rounding to midnight would give us some benefits, for example if the query fails, we know it failed for all drafts from a given date. Again, not sure if it matters to us at all.

Reply:
```
now() at time zone 'utc'
```
?

Reply:
I think the DB should be configured to default to UTC instead.